content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -281,6 +281,52 @@ Weave Net runs as a [CNI plug-in](https://www.weave.works/docs/net/latest/cni-pl
 or stand-alone.  In either version, it doesn't require any configuration or extra code
 to run, and in both cases, the network provides one IP address per pod - as is standard for Kubernetes.
 
+## IPv4/IPv6 dual stack
+
+{{< feature-state state="alpha" >}}
+{{< warning >}}Alpha features change rapidly. {{< /warning >}}
+
+IPv4/IPv6 dual stack enabled Kubernetes clusters allow the assignment of both IPv4 and IPv6 addresses simulatenously which enables the following capabilities:
+
+   * Awareness of multiple IPv4/IPv6 address assignments per Pod
+   * Native IPv4-to-IPv4 in parallel with IPv6-to-IPv6 communications to, from, and within a cluster
+
+Details on the implementation of this feature may be found in the [IPv4/IPv6 dual stack KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/20180612-ipv4-ipv6-dual-stack.md)
+
+### Supported Features
+
+Enabling IPv4/IPv6 dual stack on a Kubernetes cluster enables the following features:
+
+   * Dual stack Pod networking (multi-IP Pod)
+   * Kubenet multi address familiy support (IPv4 and IPv6)
```

Comment:
I noticed that this page doesn't have kubenet in the alphabetical list above; is that worth fixing?

Reply:
I also noticed that but feel it's out of scope of this PR. Would you be interested in fixing?